### PR TITLE
Minimized the file size of huge json files and clean some comments in notebooks

### PR DESCRIPTION
**Pull request by wnsgur4322**
check it please!
Thanks

### CODE REVIEW

**Review by laurajjiang — APPROVED** [review]

File: notebooks/vis_render/barcode.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
     <div class="left">
       <h3> Epoch #:</h3>
       <h4 id="epochNum"></h4>
-      <input class="slider" id="epoch_slider" type="range" min="0" max="19" step="1" value="0" onchange="changedValues()"/>
+      <input class="slider" id="epoch_slider" type="range" min="0" max="4" step="1" value="0" onchange="changedValues()"/>
```
Comment (thuyvyng) [inline notebooks/vis_render/barcode.html]: why is this changed to 4?

**Review by thuyvyng — COMMENTED** [review]
Screen Shot 2021-05-08 at 7 48 50 PM
Is this supposed to look like this?

File: notebooks/vis_render/barcode.html (hunk 1 above)
Comment (wnsgur4322) [reply notebooks/vis_render/barcode.html]: because I reduced the number of epoch on json file to reduce file size.

Comment (wnsgur4322) [reply notebooks/vis_render/barcode.html]: I change max value to length value from json array.

**Comment by wnsgur4322** [comment]
> Screen Shot 2021-05-08 at 7 48 50 PM
> 
> Is this supposed to look like this?

I dont know why barcode doesn't render correctly because there is no error when I debug with console message.
Even I used same code with 'barcode.html', the hub notebook works well.
